index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="index.css">
+  <title>Awesome Books</title>
+</head>
+<body>
+  <header>
+    <p class="logo">Awesome Books</p>
+    <nav class="navbar">
+      <ul>
+        <li>
+          <a href="#" id="lists">Lists</a>
+        </li>
+        <li>
+          <a href="#" id="add-books">Add new</a>
+        </li>
+        <li>
+          <a href="#" id="contact">Contact</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <span id="currentDateTime"></span>
```

Comment:
- [ ] Well done for adding the `<span>` element to display the current time! However, to fully meet the project requirements, you should use the Luxon library to implement the functionality. The [Luxon library ](https://github.com/moment/luxon/blob/master/docs/tour.md#get-the-current-date-and-time)allows you to easily work with dates and times and provides helpful formatting options.  By implementing this, your project will be in line with the specified requirements.
![image](https://github.com/78fatemanazari/Awesome-Books-ES6/assets/55840999/3b75e23b-be7b-4f60-9a2b-7b1febde16ac)

Keep up the good work and utilize the Luxon library to enhance your project! 🌟⏰🚀